src/main/java/org/broadinstitute/hellbender/engine/spark/GATKSparkTool.java

```diff
@@ -62,11 +62,6 @@
     @ArgumentCollection
     protected IntervalArgumentCollection intervalArgumentCollection = requiresIntervals() ? new RequiredIntervalArgumentCollection() : new OptionalIntervalArgumentCollection();
 
-    @Argument(doc = "maximum number of bytes to read from a file into each partition of reads. " +
-            "Setting this higher will result in fewer partitions. Note that this will not be equal to the size of the partition in memory",
-            fullName = "bamPartitionSize", shortName = "bps", optional = true)
-    protected long bamPartitionSplitSize = ReadsSparkSource.DEFAULT_SPLIT_SIZE;
-
```

Comment:
I thought about continuing to expose this, but I don't think it's necessary. I ran ApplyBQSR with 128MB partitions (on 6GB input) and it worked fine. That's not surprising as the shuffle is where we hit problems, not the map phases since the memory overhead there is very low.